### PR TITLE
beta banner removed

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**

### CODE REVIEW

File: app/views/includes/taxs_main.scala.html [hunk 1]
```diff
@@ -117,14 +117,7 @@
...
 
-@betaBanner = {
-    @uiLayouts.betaBanner(
-        userLoggedIn,
-        ApplicationConfig.betaFeedbackUrl,
-        ApplicationConfig.betaFeedbackUnauthenticatedUrl,
-        showFeedbackLinkInBetaBanner
-    )
-}
+@betaBanner = {}
```
Comment (Gaur4vGaur) [inline app/views/includes/taxs_main.scala.html]: can we get rid of the whole tag itself?

File: app/views/includes/taxs_main.scala.html [hunk 2]
```diff
@@ -117,14 +117,7 @@
...
 
 @serviceInfo = {
     @uiLayouts.serviceInfo(
```
Comment (Gaur4vGaur) [inline app/views/includes/taxs_main.scala.html]: do we still need to call beta banner from service info tag?

**Comment by vimalsuresh** [comment]
@Gaur4vGaur  could you please approve the changes made by @Muhammad-Ali-Ahmad

**Comment by Gaur4vGaur** [comment]
Please do not raise PRs without JIRA tickets

**Pull request closed without merging**